packages/core/android/src/main/kotlin/com/datadog/reactnative/DatadogSDKWrapper.kt

```diff
@@ -53,22 +77,38 @@ internal class DatadogSDKWrapper : DatadogWrapper {
     }
 
     override fun telemetryDebug(message: String) {
-        Datadog._internal._telemetry.debug(message)
+        // Do not initialize the telemetry proxy before SDK is initialized
+        if (isInitialized()) {
```

Comment:
Could you simplify this a bit and have `telemetryProxy`  return an optional, then move the check into that accessor? Same with `webViewProxy`.